flyrs/tests/test_rust_remote.py

```diff
@@ -0,0 +1,55 @@
+import subprocess
+import time
+
+from flytekit import task
+from flytekit.configuration import Config, ImageConfig, SerializationSettings
+from flytekit.remote import FlyteRemote
+from flytekit.remote.entities import FlyteTask
+
+PROJECT = "flytesnacks"
+DOMAIN = "development"
+
+TASK_NAME = "test_rust_remote.my_test_task"
+VERSION_ID = f"{hash(time.time())}"  # we use current timestamp when initialize tests, to prevent identical re-register.
+
+
+def test_register_task():
+    @task()
+    def my_test_task(n: int) -> int:
+        return n
+
+    remote_rs = FlyteRemote(Config.auto(), default_project=PROJECT, default_domain=DOMAIN, enable_rust=True)
+    flyte_task = remote_rs.register_task(
+        entity=my_test_task,
+        serialization_settings=SerializationSettings(
+            image_config=ImageConfig.auto(img_name="flyte-cr.io/image-name:tag")
+        ),
+        version=VERSION_ID,
+    )
+    assert isinstance(flyte_task, FlyteTask)
+    assert f"{flyte_task.id}" == f"TASK:{PROJECT}:{DOMAIN}:{TASK_NAME}:{VERSION_ID}"
+
+
+def test_fetch_task_without_grpc():
+    subprocess.run(["pip", "uninstall", "grpcio", "grpcio-status"])
```

Comment:
Having a test uninstall `grpcio` during a run feels like an anti-pattern. I think `flyrs` should have it's own CI environment without `grpcio` for testing.

Reply:
Sure, updated TODO list at https://github.com/flyteorg/flytekit/pull/2328#issue-2226044220